rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly
```

Comment:
Please remove this file; It seems useful at first, but becomes really confusing when your default toolchain isn't latest nightly any more, e.g. when the latest nightly is broken.